### PR TITLE
Add DimplePlus 7u Layout

### PR DESCRIPTION
**Pull request by jbaskeen**
## Description

This change introduces the 7u layout option with 2.25 enter key and 2.0 BS key to qmk for the DimplePlus keyboard. Since this layout was being added, I also updated the existing layout's layout spacing in an attempt to improve readability.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

No/A

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by jbaskeen** [comment]
@jackytrabbit FYI in case I did anything inappropriate here. This is my first qmk contribution so I'm not confident I put it together correctly for D+.

File: keyboards/lazydesigners/dimpleplus/info.json [hunk 1]
```diff
@@ -63,6 +63,60 @@
...
+                {"label":"K304 (F4,C7)", "x":4.75, "y":3.25},
+                {"label":"K305 (F4,C6)", "x":5.75, "y":3.25},
+                {"label":"K306 (F4,B6)", "x":6.75, "y":3.25},
+                {"label":"K307 (F4,B5)", "x":7.75, "y":3.25},
+                {"label":"K308 (F4,B4)", "x":8.75, "y":3.25},
+                {"label":"K309 (F4,D7)", "x":9.75, "y":3.25},
+                {"label":"K310 (F4,D6)", "x":10.75, "y":3.25, "w":1.75},
+                {"label":"K400 (F5,F1)", "x":0.75, "y":4.25},
+                {"label":"K402 (F5,F6)", "x":1.75, "y":4.25},
+                {"label":"K404 (F5,C7)", "x":3.75, "y":4.25, "w":7.00},
```
Comment (zvecr) [inline keyboards/lazydesigners/dimpleplus/info.json:116]:
```suggestion
                {"label":"K404 (F5,C7)", "x":2.75, "y":4.25, "w":7.00},
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/lazydesigners/dimpleplus/info.json (hunk 1 above)
Comment (jbaskeen) [reply keyboards/lazydesigners/dimpleplus/info.json]: @zvecr good catch, thanks. Fix pushed

**Comment by jbaskeen** [comment]
@zvecr any other feedback to get this wrapped up?

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**